lib/install/validate/storage.go

```diff
@@ -175,3 +177,34 @@ func (v *Validator) suggestDatastore(path string, label string, flag string) {
 		}
 	}
 }
+
+func parseScratchSize(input string) int {
```

Comment:
Why make life hard?  `Error` is a useful thing vs implicit and/or explicit negative values.